packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -473,13 +552,21 @@ const outputViewOptions = ref([
 	{ value: OutputView.Charts, icon: 'pi pi-image' },
 	{ value: OutputView.Data, icon: 'pi pi-list' }
 ]);
-const simulationRunResults = ref<{ [runId: string]: SimulationRunResults }>({});
+const runResults = ref<{ [runId: string]: any }>({});
+const runResultsSummary = ref<{ [runId: string]: any }>({});
 const riskResults = ref<{ [runId: string]: any }>({});
 const simulationRawContent = ref<{ [runId: string]: CsvAsset | null }>({});
 const optimizationResult = ref<any>('');
 
 const modelParameterOptions = ref<ModelParameter[]>([]);
 const modelStateAndObsOptions = ref<string[]>([]);
+
+const simulationChartOptions = computed(() => {
+	let params: string[] = [];
+	if (modelParameterOptions.value) params = modelParameterOptions.value.map((ele) => ele.id);
+
+	return [...params, ...modelStateAndObsOptions.value];
+});
```

Comment:
Maybe some clean up here as well, amongst
- modelParameterOptions
- modelStateAndObsOptions
- simulationChartOptions

In particular "modelParameterOptions" is an object array but is only used as a string-array in simulationChartOptions, so might as well make it a string-array??

Reply:
makes sense, ive made it a string array and cleaned up a little